### PR TITLE
[15.0][MIG] calendar_sms : add post-migration.py script to reload noupdate data

### PR DESCRIPTION
**Pull request by legalsylvain**
quite simple migration script.

### CODE REVIEW

File: openupgrade_scripts/scripts/calendar_sms/15.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "calendar_sms", "15.0.1.1//noupdate_changes.xml")
```
Comment (FernandoRomera) [inline openupgrade_scripts/scripts/calendar_sms/15.0.1.1/post-migration.py]: ¿ // or / in the path?

**Comment by MiquelRForgeFlow** [comment]
Please, @legalsylvain , you have a pending change. Or I would have to do the supersede that you don't like.

File: openupgrade_scripts/scripts/calendar_sms/15.0.1.1/post-migration.py (hunk 1 above)
Comment (legalsylvain) [reply openupgrade_scripts/scripts/calendar_sms/15.0.1.1/post-migration.py]: thanks. Fixed.

**Comment by legalsylvain** [comment]
> Please, @legalsylvain , you have a pending change. Or I would have to do the supersede that you don't like.

Thanks @MiquelRForgeFlow for the friendly reminder ! I had totally missed this review.

It is fixed.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**